src/app/src/main/kotlin/org/vaccineimpact/api/app/app_start/Router.kt

```diff
@@ -59,14 +59,25 @@ class Router(val config: RouteConfig,
 
         logger.info("Mapping $fullUrl to ${endpoint.actionName} on ${endpoint.controller.simpleName}")
 
-        when (endpoint.method)
+        if (contentType.contains("json"))
         {
-            HttpMethod.get -> Spark.get(fullUrl, contentType, route, this::transform)
-            HttpMethod.post -> Spark.post(fullUrl, contentType, route, this::transform)
-            HttpMethod.put -> Spark.put(fullUrl, contentType, route, this::transform)
-            HttpMethod.patch -> Spark.patch(fullUrl, contentType, route, this::transform)
-            HttpMethod.delete -> Spark.delete(fullUrl, contentType, route, this::transform)
-            else -> throw UnsupportedValueException(endpoint.method)
+            when (endpoint.method)
+            {
+                HttpMethod.get -> Spark.get(fullUrl, contentType, route, this::transform)
+                HttpMethod.post -> Spark.post(fullUrl, contentType, route, this::transform)
+                HttpMethod.put -> Spark.put(fullUrl, contentType, route, this::transform)
+                HttpMethod.patch -> Spark.patch(fullUrl, contentType, route, this::transform)
+                HttpMethod.delete -> Spark.delete(fullUrl, contentType, route, this::transform)
+                else -> throw UnsupportedValueException(endpoint.method)
+            }
+        }
+        else
+        {
+            when (endpoint.method)
```

Comment:
For all other methods returning csv data we've used the `StreamSerializable` construction which bypasses the transform and writes the content directly to the output stream. In this case that whole construction would be overkill, just want to return a raw string with no transformation, which apparently we haven't done before.